e2e/tests/tickets.e2e.ts

```diff
@@ -0,0 +1,140 @@
+import {device, by, element, expect} from 'detox';
+import {goToPage} from "../utils/common";
+import {skipOnboarding} from "../utils/onboarding";
+import setLocation from "../utils";
+
+describe('Tickets', () => {
+    beforeAll(async () => {
+        await device.launchApp({
+            permissions: {
+                location: 'inuse',
+            },
+            languageAndLocale: {
+                language: 'en',
+                locale: 'US',
+            },
+        });
+        await setLocation(62.4305, 9.3951);
+        await skipOnboarding();
+
+        // Accept ticket limitations
+        await goToPage('Tickets')
+        await expect(element(by.text('Try buying tickets'))).toBeVisible();
+        await element(by.text('Accept')).tap()
+    });
+
+    beforeEach(async () => {
+        await device.reloadReactNative();
+    });
+
+    it('should be able to buy a single ticket', async () => {
+        const ticketPrice = '42 kr'
+
+        await goToPage('Tickets')
+
+        await expect(element(by.text('Buy'))).toBeVisible();
+        await expect(element(by.text('Valid'))).toBeVisible();
+        await expect(element(by.text('New single ticket'))).toBeVisible();
+        await expect(element(by.text('New period ticket'))).toBeVisible();
+
+        // Choose single ticket
+        await element(by.text('New single ticket')).tap()
+
+        await expect(element(by.text('Single ticket'))).toBeVisible();
+        await expect(element(by.text('1 Adult'))).toBeVisible();
+        await expect(element(by.text('Travel in 1 zone (A)'))).toBeVisible();
+        await expect(element(by.text(`Total: ${ticketPrice}`))).toBeVisible();
+        await expect(element(by.text('Go to payment'))).toBeVisible();
+
+        await element(by.text('Go to payment')).tap();
+
+        await expect(element(by.text('Single ticket'))).toBeVisible();
+        await expect(element(by.label(`1 Adult ${ticketPrice} Single ticket bus/tram Valid in zone A Starting now`)).atIndex(0)).toExist()
+        await expect(element(by.text('1 Adult'))).toBeVisible();
+        await expect(element(by.text(ticketPrice)).atIndex(0)).toBeVisible();
+        await expect(element(by.text('Single ticket bus/tram'))).toBeVisible();
+        await expect(element(by.text('Valid in zone A'))).toBeVisible();
+        await expect(element(by.text('Starting now'))).toBeVisible();
+
+        /*
+        TODO How to check that 'Confirm option' is disabled before choosing a payment method?
+        await expect(element(by.text('Confirm option'))).not.toBeFocused()
+        await element(by.text('Confirm option')).getAttributes()
+            .then(e => {
+                if (!('elements' in e)) {
+                    console.log('Confirm option: ', e.text)
+                    console.log('Confirm option: ', e.value)
+                    console.log('Confirm option: ', e.enabled)
```

Comment:
Hmm lurer på om `enabled` kan være rett verdi å sjekke, men må sikkert velge "rett" element å sjekke siden det kommer an på hva som sjekkes. Hvis det sjekkes et tekst-element så får en sikkert ikke rett verdi, men sjekker en noe som er interactable (type knapp/touchable) så kanskje?

Reply:
Kan prøve på nytt nå da du har lagt til test-id på knappen. Mulig jeg får mer info ut av dette elementet.

Reply:
Fikk ikke mer info ut av denne, men fant en vei rundt ved å tap-e knappen og sjekke at ingenting skjer. Greit for nå.